### ISSUE TITLE
default_body_scrubber Regex.match? call fails if property is non-string.

### ISSUE THREAD

**Issue by begedin** [getsentry/sentry-elixir#96]
With, for example, a payload of {"some_property": false}, the `default_body_scrubber` will fail with the following stacktrace:

```
Request: POST /some/endpoint
** (exit) an exception was raised:
    ** (FunctionClauseError) no function clause matching in Regex.match?/2
        (elixir) lib/regex.ex:159: Regex.match?(~r/^(?:\d[ -]*?){13,16}$/, false)
        (sentry) lib/sentry/plug.ex:149: anonymous fn/1 in Sentry.Plug.default_body_scrubber/1
        (elixir) lib/enum.ex:1188: anonymous fn/3 in Enum.map/2
        (stdlib) lists.erl:1263: :lists.foldl/3
        (elixir) lib/enum.ex:1188: Enum.map/2
        (sentry) lib/sentry/plug.ex:146: Sentry.Plug.default_body_scrubber/1
        (sentry) lib/sentry/plug.ex:107: Sentry.Plug.build_request_interface_data/2
```

Reason being, `Regex.match?` expects a string as the second argument, which a boolean clearly is not. Same goes for any other non-string, ex. integer

**Comment by mitchellhenke** [comment]
Oops, good catch.  Thanks for the report.  🙂 

It's been fixed in https://github.com/getsentry/sentry-elixir/commit/fe55fccecbfc62447f4dedb4cf3cdd36597b8ebf and released in `2.0.2`

**Comment by JoshSmith** [comment]
Ah good job reporting this @begedin. I saw this same issue but figured it was a "me" error.
